### DISCUSSION TITLE
Difference between version 0.14.6 and 0.15.2

### DISCUSSION THREAD

**Discussion by dirajag1** [submariner-io/submariner#2676] [Q&A]
Hello,

Whats is the difference between these 2 submariner versions ? Which of these version will be supported on OCP versions 4.12 and (or) 4.14 with OVN-Kubernetes ?

https://submariner.io/community/releases/#v0146-july-5-2023
https://submariner.io/community/releases/#v0152-july-4-2023

Regards,
Divya

**Comment by skitt** [comment]
0.14.6 is the latest release on the 0.14 branch, which uses components compatible with Kubernetes 1.25 (with support back to 1.17).
0.15.2 is the latest release on the 0.15 branch, which uses components compatible with Kubernetes 1.26 (with support back to 1.19).
The 0.15 branch adds a number of features compared to the 0.14 branch, notably support for dual-stack IPv4/IPv6 setups.

Both versions support OVNKubernetes, and should work with OpenShift 4.12 and 4.14 (but the latter is untested, as far as I’m aware). For OVN-IC support on 4.14 you’ll have to wait for 0.16. As a general rule, unless you have a particular reason to stick to an older release branch, you should always use the latest release, which is currently 0.15.2.

**Reply by dirajag1** [reply]
Thanks we will test out 0.15.2
